mobile-sdk/ios.md

```diff
@@ -0,0 +1,464 @@
+
+To forward the custom-scheme payment urls to the SDK, implement the [`application(_:open:options:)`][uiappdelegate-openurl] method in your application delegate, and call `SwedbankPaySDK.open(url: url)` to let the SDK handle the url.
+
+```swift
+    func application(
+        _ app: UIApplication,
+        open url: URL,
+        options: [UIApplication.OpenURLOptionsKey : Any] = [:]
+    ) -> Bool {
+        return SwedbankPaySDK.open(url: url)
+    }
+```
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant Page as 3rd Party Page
+    participant Safari
+    participant Merchant as Payment Url Host
+    participant Trampoline as Payment Url Trampoline
+    participant App as Application
+    participant SDK
+
+    Page ->> Safari: Navigate to payment url (at Merchant)
+    Safari ->> Merchant: GET <payment url>
+    Merchant ->> Safari: 301 Moved Permanently Location: <redirect url>
+    Safari ->> Trampoline: GET <redirect url>
+    Trampoline ->> Safari: <http>...<a href="<payment url with fallback flag>">...</http>
+    Safari ->> User: Page with "Back to Application" button
+    User ->> Safari: Tap on button
+    Safari ->> Merchant: GET <payment url with fallback flag>
+    Merchant ->> Safari: 301 Moved Permanently Location: <custom scheme url>
+    Safari ->> User: Confirmation Dialog
+    User ->> Safari: Accept App Launch
+    Safari ->> App: Bring to foreground
+    Safari ->> App: application(application, open: url)
+    App ->> SDK: SwedbankPaySDK.open(url: url)
+    SDK ->> SDK: Reload payment menu
+```
+
+{% include iterator.html prev_href="android"
+                         prev_title="Back: Android"
+                         next_href="flowcharts"
+                         next_title="Next: Process Diagrams" %}
+
+[cocoapods]: https://cocoapods.org/
+[cocoapods-gettingstarted]: https://guides.cocoapods.org/using/getting-started.html
+[cocoapods-using]: https://guides.cocoapods.org/using/using-cocoapods.html
+[payment-url]: /checkout/payment-menu#payment-url
+[custom-scheme-1]: /assets/img/mobile-sdk/ios-custom-scheme-1.png
+[custom-scheme-2]: /assets/img/mobile-sdk/ios-custom-scheme-2.png
+[assoc-domains-entitlement]: /assets/img/mobile-sdk/ios-assoc-domain.png
+[ios-custom-scheme]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app
+[ios-universal-links]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content
+[ios-universal-links-routing]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content#3001753
+[ios-custom-scheme]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app
```

Comment:
Codacy found an issue: [[no-duplicate-definitions] Do not use definitions with the same identifier (454:1)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5758572)